lapce-proxy/src/dispatch.rs

```diff
@@ -43,7 +43,6 @@ use crate::{
 
 const OPEN_FILE_EVENT_TOKEN: WatchToken = WatchToken(1);
 const WORKSPACE_EVENT_TOKEN: WatchToken = WatchToken(2);
-
 pub struct Dispatcher {
```

Comment:
don't delete the line here

Reply:
my OCD in shambles. I will try to include this revert with the Cargo.lock changes removal.

Reply:
`git checkout -- Cargo.lock`